packages/protocol/contracts/governance/Election.sol

```diff
@@ -431,10 +433,18 @@ contract Election is
       votes.active.total
     );
     FixidityLib.Fraction memory score = FixidityLib.wrap(
-      getValidators().calculateGroupEpochScore(uptimes)
+      validators.calculateGroupEpochScore(uptimes)
+    );
+    FixidityLib.Fraction memory slashingMultiplier = FixidityLib.wrap(
```

Comment:
We call it `penalty` in the contract but `slashingMultiplier` here. I have no strong feelings (a slight preference for `slashingMultiplier`), but let's be sure to be consistent.